connectors/easy-post/element-templates/easy-post-connector.json

```diff
@@ -0,0 +1,943 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Easy Post Connector",
+  "id": "io.camunda.connectors.EasyPost.v1",
+  "version": 1,
+  "description": "Orchestrate your Easy Post shipment with Camunda",
```

Comment:
orchestrate doesn't fit in this context.
Maybe sth like this: It allows you to create addresses, parcels, and shipments, as well as purchase and verify shipments.